README.md

```diff
@@ -168,6 +171,37 @@ attribution:
     - john@opensauced.pizza
 ```
 
+### 🚀 New in v1.4.0: Generate Config
```

Comment:
This doesn't make alot of sense in the README: seems better suited for a changelog. Could we instead position this to have a "quick start" section after the install instructions that walks through

1. Generating a config
2. running codeowners

We'll want to point people to the `docs/pizza.md` and rely on that for documenting how things work and more advanced use cases. Otherwise, this README will get really unwieldy.

We can also get this in and I can take a stab at the README since I think there's a few other things I wanted to refactor (like the config definition in the README)

Reply:
In general, I agree. For the launch, I thought it would make sense to put it here only to help increase usage since this seems to be the biggest point of friction. 

I think we decided in the engineering sync that the Quick Start would go in the docs. Do you want me to move it there? At some point, I thought we were deprioritizing docs and focusing on getting everything in here. But maybe I misunderstood.

Reply:
> I thought it would make sense to put it here only to help increase usage since this seems to be the biggest point of friction.

Very true!

> I think we decided in the engineering sync that the Quick Start would go in the docs.

You're right: I'm probably waffling trying to find the best way to go about this. CLIs are really hard to tell people about and get them to grok without getting their hands dirty.